docs/modules/operation/pages/provisioning/detectors/Win32ServiceDetector.adoc

```diff
@@ -0,0 +1,34 @@
+= Win32 Service Detector
+
+Use the Win32 Service Detector to detect the availability of services on Windows devices with SNMP enabled.
+It will query against the Microsoft LAN Manager MIB `.1.3.6.1.4.1.77` over SNMP to confirm the service.
```

Comment:
It will query the table `.1.3.6.1.4.1.77.1.2.3.1.3` (svSvcOperatingState)  from the LAN manager MIB-II and if the service is active(1), the service will be detected.
Each character of the display name of the service is converted into the decimal representation of the ASCII character and is used as an OID suffix for the _svSvcOperatingState_ table to identify its state.
The display name _Task Scheduler_ becomes OID suffix `.84.97.115.107.32.83.99.104.101.100.117.108.101.114`.

TIP: You can query the `1.3.6.1.4.1.77.1.2.3.1.1` (svSvcName) to figure out which display name maps to the exact OID which can be used later in the detector to query the service state.